index.js

```diff
@@ -104,7 +106,12 @@ function finalhandler(req, res, options) {
       return req.socket.destroy()
     }
 
-    send(req, res, status, msg)
+    if(typeof render === 'function'){
```

Comment:
Calling `finalhandler(req, res, {render: 42})` needs to throw an error, not silently pretend like the option was not there.